src/index.js

```diff
@@ -22,6 +22,8 @@ import Tab from './components/Tab.react';
 import Store from './components/Store.react';
 import LogoutButton from './components/LogoutButton.react';
 
+import './components/css/react-dates@12.3.0.css';
```

Comment:
Moving this style dependency to the top level -- jest is not able to handle css files correctly atm